build-dev.xml

```diff
@@ -16,7 +16,7 @@
     <target name="standards" depends="phplint,ecs,phpstan,twig-lint,yaml-lint,eslint-check" description="Checks coding standards."/>
```

Comment:
you need to be more descriptive in commit message "phing target: Make couple of changes" - I do not know what happened and why :wink: